src/style.css

```diff
@@ -1 +1,88 @@
-/* Styles go here */
+:root {
+  --blueHover: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
```

Comment:
![image](https://github.com/mate-academy/layout_moyo-header/assets/59209337/e2b7ee27-5acc-4992-9796-ec4ad26d8073)

Reply:
Hello, did you mean that in my case it is not necessary to use quotation marks? Or I need to add one more alternative fonts, except sans-serif?

Reply:
Hello, if you have any questions about how to resolve the comments, feel free to ask it in the fe_chat, instead of re-requesting review
no, i mean you should use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)

Reply:
I found that the best fallback fonts for "Roboto" are "Helvetica Neue" and "sans-serif". Is it a good decision?)